tdrs-frontend/src/components/SplashPage/SplashPage.jsx

```diff
@@ -28,6 +28,9 @@ function SplashPage() {
       // This doesn't need to be tested, it will never be reached by jest.
       event.preventDefault()
       dispatch(setMockLoginState())
+    } else if (process.env.REACT_APP_LOGIN_XMS_URL) {
+      event.preventDefault()
+      window.location.href = process.env.REACT_APP_LOGIN_XMS_URL
```

Comment:
```suggestion
```suggestion
    } else if (process.env.LOGIN_XMS_USED) {
      event.preventDefault()
      window.location.href = `${process.env.REACT_APP_BACKEND_URL}/login/xms`
```
```